src/getSearchKeywords.js

```diff
@@ -0,0 +1,53 @@
+function getSearchKeywords() {
```

Comment:
import 해서 사용할 수 있도록 export가 필요할 것 같아요
```suggestion
export function getSearchKeywords() {
```